### DISCUSSION TITLE
We need Tauri mode!

### DISCUSSION THREAD

**Discussion by Edge-coordinates** [quasarframework/quasar#17077] [Ideas / Proposals]
It would be nice to have tauri mode in Quasar (Quasar dev -m tauri)

Tauri is an excellent framework, and version 2.0 has already been released in beta preview. Despite being built with Rust, it offers a rich JavaScript interface, so you can even build applications using just JavaScript. Its final software size is much smaller than Electron's and offers high performance.

At the same time, they are providing support for mobile platforms, making it a truly cross-platform framework.

I believe this support should be included in the official plan rather than relying on community-maintained templates. Thank you!

**Comment by ajmakoni** [comment]
I have been doing some reading on Tauri and I think it will be a great fit with Quasar framework.
Some consideration on this matter by Quasar Team will be greatly appreciated!!

**Comment by mr-zwets** [comment]
a Tauri mode would be awesome 💯
